README.md

```diff
@@ -72,7 +72,13 @@ make build
 ### Running
 
 Quick start a NEO node on the private network. This requires the [neo-privatenet](https://hub.docker.com/r/cityofzion/neo-privatenet/) Docker image running on your machine.
```

Comment:
We're on master branch here, you can drop neo-privatenet completely from the README.

Reply:
Fixed.